network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgWriter.java

```diff
@@ -944,6 +948,9 @@ private void addMetadata(Graph graph, XMLStreamWriter writer) throws XMLStreamEx
                                   String.valueOf(svgParameters.getPowerValuePrecision()), String.valueOf(svgParameters.getAngleValuePrecision()),
                                   String.valueOf(svgParameters.getCurrentValuePrecision()), svgParameters.getEdgeInfoDisplayed().name(),
                                   getFormattedValue(svgParameters.getPstArrowHeadSize()), svgParameters.getUndefinedValueSymbol());
+        metadata.addLayoutParameters(String.valueOf(layoutParameters.isTextNodesForceLayout()), getFormattedValue(layoutParameters.getSpringRepulsionFactorForceLayout()),
```

Comment:
using `getFormattedValue` makes sense when you're writing metadata about x/y coordinates which are also in the svg, as in the svg the coordinates are written with a fixed precision thanks to this method. But I don't think you should use it for all other values, especially the one which might need more precision (spring repulsion factor, fixed scale, the angles in radians loopEdgesAperture and edgesForkAperture, ...)

Reply:
Put original values instead of rounded formatted values for parameters in metadata